mathics/builtin/numbers/algebra.py

```diff
@@ -587,21 +601,24 @@ def coeff_power_internal(self, expr, var_exprs, filt, evaluation, form="expr"):
             var_pats = [Pattern.create(var) for var in var_exprs]
 
         # ###### Auxiliary functions #########
-        def key_powers(lst):
+        def key_powers(lst: list) -> int:
             key = Expression(SymbolPlus, *lst)
             key = key.evaluate(evaluation)
             if key.is_numeric(evaluation):
                 return key.to_python()
```

Comment:
Or maybe a comment or an assertion here if this is supposed to always be an integer.

Reply:
Indeed, it could be a float...